lib/util/manifest_parser_utils.js

```diff
@@ -293,6 +293,7 @@ shaka.util.ManifestParserUtils.AUDIO_CODEC_REGEXPS_ = [
   /^[ae]c-3$/,
   /^ac-4$/,
   /^dts[cex]$/, // DTS Digital Surround (dtsc), DTS Express (dtse), DTS:X (dtsx)
+  /^iamf/,
```

Comment:
Perhaps we should also add an iamf mimeType to `shaka.media.MediaSourceEngine.probeSupport`?

Reply:
probeSupport is only for the Demo, so it’s not necessary